packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -69,19 +64,23 @@ const JurisdictionModal = () => {
                     <Modal.Header onRequestClose={closeModal}>
                         <Text weight='bold'>{jurisdictionTitle}</Text>
                     </Modal.Header>
-                ) : null}
+                ) : (
+                    <DynamicLeverageTitle />
+                )}
                 <Modal.Body
                     className={twMerge(
-                        'p-0 flex flex-col relative min-h-0 overflow-auto lg:w-auto lg:h-auto lg:p-8 lg:min-w-[1100px]'
+                        'p-0 flex flex-col relative min-h-0 overflow-auto w-screen h-screen lg:w-auto lg:h-auto lg:p-8 lg:min-w-[1100px]',
+                        isDynamicLeverageVisible && 'lg:min-h-[700px]'
                     )}
                 >
-                    {isDynamicLeverageVisible && <DynamicLeverageTitle />}
                     <JurisdictionScreen setIsCheckBoxChecked={setIsCheckBoxChecked} />
                     {isDynamicLeverageVisible && <DynamicLeverageScreen />}
-                    <JurisdictionTncSection
-                        isCheckBoxChecked={isCheckBoxChecked}
-                        setIsCheckBoxChecked={setIsCheckBoxChecked}
-                    />
+                    {!isDynamicLeverageVisible && (
+                        <JurisdictionTncSection
+                            isCheckBoxChecked={isCheckBoxChecked}
+                            setIsCheckBoxChecked={setIsCheckBoxChecked}
+                        />
+                    )}
                 </Modal.Body>
                 {!isDynamicLeverageVisible ? (
                     <Modal.Footer className='lg:rounded-default'>
```

Comment:
The top-border of footer is removed in sm+md view 
<img width="1243" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/52b5264b-75eb-4b95-938a-fea5fb139608">